src/css/base/variables/_colors.scss

```diff
@@ -1,102 +1,209 @@
+  --color-typography-breadcrumbs-highlight: var(--color-royal-blue);
+  --color-typography-bullet: var(--color-cadet-blue);
+  --color-typography-button: var(--color-martinique);
+  --color-typography-code-line-number: var(--color-blue-bayoux);
+  --color-typography-code: var(--color-ebony-clay);
+  --color-typography-content-navigation-text: var(--color-ebony-clay);
+  --color-typography-content-navigation-title-colored: var(--color-piction-blue);
+  --color-typography-dropdown: var(--color-ebony-clay);
+  --color-typography-hero-cta: var(--color-royal-blue);
+  --color-typography-footer-copyright: var(--color-dusty-gray);
+  --color-typography-footer-icon: var(--color-black);
+  --color-typography-footer-link: var(--color-mine-shaft);
+  --color-typography-icon-checkmark: var(--color-piction-blue);
+  --color-typography-last-updated: var(--color-san-marino);
+  --color-typography-label: var(--color-white);
+  --color-typography-logo: var(--color-martinique);
+  --color-typography-on-this-page: rgba(35, 38, 59, 0.75); /* Equivalent of transparentize($color-ebony-clay, 0.25) */
+  --color-typography-panel-grid-link: var(--color-ebony-clay);
+  --color-typography-sidebar-button: var(--color-mine-shaft);
+  --color-typography-table-body: var(--color-black);
+  --color-typography-table-header: var(--color-ebony-clay);
+  --color-typography-tabs-title-active: var(--color-dodger-blue);
+  --color-typography-tabs-title: var(--color-black-soft);
+  --color-typography-toc-link-active: var(--color-ebony-clay);
+  --color-typography-toc-link: var(--color-cadet-blue);
+  --color-typography-topic-grid-text: var(--color-san-marino);
+  --color-typography-topic-grid-title: var(--color-ebony-clay);
+  --color-typography-topic-link: var(--color-piction-blue);
+  --color-typography-topic-text: var(--color-black);
+  --color-typography-topic-title: var(--color-ebony-clay);
+  --color-typography-search-box-input: rgba(80, 80, 80, 0.5);
+  --color-typography-search-box-input-placeholder: rgba(80, 80, 80, 0.5) !important;
+  --color-typography-admonition-caution: var(--color-web-orange);
+  --color-typography-admonition-note: var(--color-denim);
+  --color-typography-admonition-tip: var(--color-apple);
+  --color-typography-admonition-warning: var(--color-cinnabar);
+
+  /* Color Filter */
+  --color-filter-sidenav-toctree: invert(38%) sepia(71%) saturate(6789%) hue-rotate(231deg) brightness(90%) contrast(95%);
+
+}
+
+
+.dark {
+  --dark-navigation-base: rgba(251, 251, 255, 0.04);
+  --dark-navigation-one: rgba(22, 22, 22, 0.04);
+  --dark-navigation-base-solid: rgb(41, 45, 62);
+  --dark-navigation-solid: #16212B;
+  --dark-typography-title: #F5F5FF;
+  --dark-typography-title-link: rgba(36, 213, 242, 0.9);
+  --dark-typography-paragraph: #9C9C9C;
+  --dark-surface-base: #09090A;
+  --dark-surface-one: #161616;
+  --dark-surface-two: #1A1A1B;
+  --dark-surface-three: #1C1C1D;
+  --dark-surface-four: #1F1F20;
+  --dark-surface-five: #242425;
```

Comment:
We could define colors base colors under root, using the name from https://chir.ag/projects/name-that-color

This way we have the base complete color palette under the same place:

```
:root {
  /* Base Colors */
  --color-alto: #e0e0e0;
  --color-apple: #43a047;
  --color-black: #000;
 (...)
 ```